.github/workflows/stale.yml

```diff
@@ -0,0 +1,32 @@
+name: Mark stale pull requests
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v9
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        start-date: '2024-04-19T00:00:00Z' # ISO 8601 or RFC 2822
+        stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 14 days.'
+        stale-pr-message: 'This PR is stale because it has been open for 30 days with no activity. Remove stale label or comment or this will be closed in 14 days'
+        close-issue-message: 'This issue was closed because it has been stalled for 14 days with no activity.'
+        close-pr-message: 'This PR was closed because it has been stalled for 14 days with no activity.'
+        stale-pr-label: 'stale'
+        exempt-issue-labels: 'good-first-issue'
+        exempt-all-milestones: true
+        days-before-stale: 30
+        days-before-close: 14
+
+  unassign:
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Auto unassign stale assignments
+      uses: JasonEtco/slash-assign-action@v1
+      with:
+        required_label: 'good-first-issue'
```

Comment:
It's mean every day will unassign for `good-first-issue`? and not `good-first-issue` & `stale`? could you like to explain here?

Reply:
It means that this workflow will only search those issues with good-first-issue labels.

Reply:
Yes, so it's  with `JasonEtco/slash-assign-action` for stable assign,  different with `actions/stale`. found the answer from https://github.com/JasonEtco/slash-assign-action?tab=readme-ov-file#days_until_unassign